docs/Manuals/client-manual.rst

```diff
@@ -1547,17 +1556,22 @@ has been selected.
   Unit Controls Widget
 
 
-You can see that the :unit:`Workers` is selected because it has a white selection ring around its base. Looking
-at the dialog, in the header, you can see that this unit is ID # 111, has 4 3/9 Move Points (MPs), and 10 of
-10 Hit Points (HPs). From left to right you can see an image of the unit with MPs overlaid, the terrain it is
-on with infrastructure improvements shown, and then lastly the actions that this unit can take. In this
-example the actions available are: Plant to Forest/River, Build Road, Go to Tile, Sentry, Auto Worker, Wait,
-and Done.
+You can see that the :unit:`Workers` is selected because it has a white selection ring around its base.
+Looking at the dialog, in the header, you can see that this unit is ID # 111, has 4 3/9 Move Points (MPs), and
+10 of 10 Hit Points (HPs).
```

Comment:
Do we need to specify that Move Points are MPs and Hit Points are HPs?

Reply:
It's typically good form to document want an acronym is, especially when shown on an image. Don't want to assume the reader knows what they mean.